gradle-failure-reports-exceptions/src/main/java/com/palantir/gradle/failurereports/exceptions/FailureReporterException.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.failurereports.exceptions;
+
+import com.palantir.gradle.failurereports.common.FailureReport;
+
+/**
+ * Superclass of RuntimeException that can display the exception as a Failure Report in Circle CI.
```

Comment:
not important rn, but curious what differences there'd be for github actions

Reply:
even bigger nit - subclass, not superclass

Reply:
> not important rn, but curious what differences there'd be for github actions

the github actions written as a hackweek project or are you referring to something else ? If it is the former, then it will be working as expected without any extra work (I am rendering all the FailureReports into another file that can be read by github actions: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#example-creating-an-annotation-for-an-error)

Reply:
> even bigger nit - subclass, not superclass

ups, yes! 🤦‍♀️